app/src/main/res/layout/meal_item_view.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="@dimen/x_large_300"
+    android:layout_height="wrap_content"
+    android:clipToPadding="false">
+
+    <androidx.cardview.widget.CardView
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/x_large_350"
+        android:backgroundTint="@color/white"
+        android:elevation="@dimen/small_50"
+        app:cardCornerRadius="@dimen/normal_200"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <de.hdodenhof.circleimageview.CircleImageView
+        android:id="@+id/meal_image"
+        android:layout_width="@dimen/small_0"
+        android:layout_height="@dimen/small_0"
+        android:layout_centerHorizontal="true"
+        android:layout_marginBottom="@dimen/x_large_200"
+        android:elevation="@dimen/small_135"
+        android:src="@drawable/ic_launcher_background"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHeight_percent="0.5"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintWidth_percent="0.65" />
+
+    <LinearLayout
```

Comment:
Why do we need this linear layout here? From what i can see, it only wraps a textview